### DISCUSSION TITLE
Boot Freeshow with forced output to current monitor?

### DISCUSSION THREAD

**Discussion by MrShinyBoots** [ChurchApps/ChurchAppsSupport#329] [FreeShow] [answered]
I know that I can use the present button then choose to force output to the current monitor - is there a way to initiate this behaviour at the start of the program - if not would it be possible to add this option to settings?

Usage - small form factor pc / Rpi Booting into freeshow and starting as a presentation for stage usage (output force could be either stage or general output - ideally selectable) - this would save ram if I'm presenting on one screen and running with freeshow at boot as i wouldn't have to also boot another web browser to then connect to itself - this may also improve the speed of the presentation. - of course the app could then be controlled with remote etc - and you could always escape out of present mode.

many thanks

**Comment by vassbo** [comment]
I will see what I can do.

**Comment by vassbo** [comment, marked as answer]
I managed to add a toggle; "Allow displaying output over main screen" in the "Change output values" screen. This means you should be able to do this in the next update.
